tests/cases/helpers/case_helpers_es_care_platform.yaml

```diff
@@ -144,4 +144,99 @@ ESCarePartnerSendMessageToSenior:
       Id: $PAGE.care_platform_participant_records.message_submit_button$
     # Log out.
     - Type: case
-      Value: CarePlatformLogout
\ No newline at end of file
+      Value: CarePlatformLogout
+
+ESSetScheduledCallForSeniors:
```

Comment:
This helper should mention that it's a care partner helper in the name

Reply:
updated the name on this commit https://github.com/neveralone-chs-org/TestRay/pull/117/commits/b644eb0d9d8b9615cbb6200c79c81cbfb06e8293